test/fluree/db/query/fql_test.clj

```diff
@@ -139,8 +130,7 @@
         people (test-utils/load-people conn)
         db     (fluree/db people)]
     (testing "with 2 separate fn binds"
-      (let [q   '{:context  {:ex "http://example.org/ns/"}
```

Comment:
Maybe I missed it, but it looks like this PR removes all the uses of context in queries? 

If that's the case, it'd be helpful to modify some of these and/or add a couple of tests so that we _do_ exercise query contexts somewhere. Tbh I'm not sure if all of these tests were really using those contexts in any meaningful way in the first place, but I think it'd be good to have some tests that do

Reply:
Correct, some of the tests were not using context in an idiomatic way anyhow.

I did leave some tests which specify a context with a query, where the query and/or results didn't depend on the default context at all. So we do have some, but agree more is better!

Worth emphasizing your point, now if you supply a `:context` it won't use the default context at all. The default context is truly a "default". As mentioned above, there is syntax to merge the default context into your context - but nothing magic will ever happen without your instruction.

Reply:
> I did leave some tests which specify a context with a query, where the query and/or results didn't depend on the default context at all. So we do have some, but agree more is better!

Ahh yes I see them now, great!

> but nothing magic will ever happen without your instruction.

Yeah, I'm very glad to see a reduction in magic 😄